AY2021S1-CS2103-T16-3/tp
# Add Aliasing support to ResiReg

### What this does
This PR adds aliasing to ResiReg, namely 3 commands: 
- `alias` : Adds an alias. Format: `alias c/<command> a/<alias>` e.g. `alias c/rooms a/r`
- `dealias` : Deletes an alias. E.g. `dealias c/rooms a/r`
- `aliases` : Lists all aliases. Format: `aliases`

### How to test

1. Launch the application.
2. Type in the above 3 commands. 
3. Note that you can't have the same alias for different commands, an alias cannot be a command word, and undo/redo don't apply to alias commands (because they do not change the state of ResiReg). 

### Notes
Added a lot of testing in this PR. Note to team-mates: please add Integration tests for the commands you've added to ResiReg by this weekend - it takes a LOT of work. 

Fixes #103
